docs/serving/samples/hello-world/helloworld-go/README.md

```diff
@@ -72,27 +72,28 @@ cd knative-docs/docs/serving/samples/hello-world/helloworld-go
    # Use the offical Golang image to create a build artifact.
    # This is based on Debian and sets the GOPATH to /go.
    # https://hub.docker.com/_/golang
-   FROM golang:1.12 as builder
-
+   FROM golang:1.13 as builder
+   
```

Comment:
Extra whitespace (here and in a few other places).

Reply:
The Dockerfile spacing and comments are designed to emphasize a pipeline of steps and provide context on how the container is assembled.

Reply:
Ah, README whitespace. Addressing.